tests/active_features/wp-cli.feature

```diff
@@ -16,7 +16,9 @@ Feature: Running WP-CLI Commands on a Drupal Site
   @vcr remote-wp.yml
   Scenario: Running a WP-CLI command that is not permitted
     When I run: terminus wp [[test_site_name]].dev -- db query 'CHECK TABLE $(wp db tables | paste -s -d',');'
-    Then I should get:
-    """
-    That command is not available via Terminus. Please use the native wp command.
-    """
+    Then I should see an error message: That command is not available via Terminus. Please use the native wp command.
+
+  @vcr remote-drush.yml
+  Scenario: Running a WP-CLI command on a Drupal site is not possible
+    When I run: terminus wp [[test_site_name]].dev -- cli version
+    Then I should see an error message: The wp command is only available on a sites running wordpress. The framework for this site is drupal8.
```

Comment:
Ahhh. Thanks for clarifying. Good catch.
Where are you on a scale 1-10 of better messages through framework labels? I've almost got an interface I like but I am like 5/10. Nice but not critical. I think I would drop it if you were less enthusiastic.